python/ray/rllib/tests/test_ignore_worker_failure.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import gym
+import unittest
+
+import ray
+from ray.rllib import _register_all
+from ray.rllib.agents.registry import get_agent_class
+from ray.tune.registry import register_env
+
+
+class FaultInjectEnv(gym.Env):
```

Comment:
Can you wrap this in a builder? Pytest will hang because with high probability, it won't find this class.

Reply:
We don't use pytest for rllib